mkdocs.yml

```diff
@@ -14,6 +14,7 @@ pages:
   - 'Welcome to St. Jude Cloud': 'index.md'
   - 'Create An Account': 'create-an-account.md'
   - 'Data Guides': 
+      - 'Studies': 'guides/data/studies.md'
       - 'Types of Data': 'guides/data/types-of-data.md'
```

Comment:
Is this the order it shows in? It needs to go under "types of data" and above "metadata"